calculate.py

```diff
@@ -1,22 +1,30 @@
 import sys
 import re
-from nparser import Parser
+from parse import Parser

```

Comment:
just a small detail but do you think parser will be more correct name?

Reply:
Having the file name be the same as the class name makes importing complicated. I think that that is why the original file name had an n prefixed to it.